openff/bespokefit/cli/prepare.py

```diff
@@ -0,0 +1,62 @@
+import json
+
+import click
+from click import ClickException
+from pydantic import ValidationError
+
+from openff.bespokefit.optimizers import get_optimizer
+from openff.bespokefit.schema.fitting import (
+    BespokeOptimizationSchema,
+    OptimizationSchema,
+)
+
+_SCHEMA_TYPES = {"general": OptimizationSchema, "bespoke": BespokeOptimizationSchema}
+
+
+@click.command("prepare")
+@click.option(
```

Comment:
Should this be an argument as it is required to run? This would also make the command slightly simpler as we would not need the `-i` flag?